test/test_torch.py

```diff
@@ -9841,6 +9841,25 @@ def test_add(self, device):
                                r"For integral input tensors, argument alpha must not be a floating point number\.",
                                lambda: torch.add(m1, m2, alpha=1.0))
 
+        # Scalar
+        for dtype in torch.testing.get_all_dtypes():
```

Comment:
why are there python tests in this PR?  AFAIK this PR doesn't have any effect on python -- shouldn't these tests be on the the PR that changes the python binding and shouldn't we have C++ tests here?

Reply:
They will take effect after the stacked PR is merged

Reply:
right, but they should be on the PR that makes the change, not on the prereq PR.

Reply:
I have moved the test code to the next PR.